Cargo.toml

```diff
@@ -54,6 +55,11 @@ futures = "0.3.5"
 pin-project = "0.4.22"
 serde_with_macros = "1.1.0"
 
+sqlx = { version = "0.4.0-beta.1", optional = true, default-features = false, features = [
+        "runtime-tokio",
+        "macros",
+        "sqlite",
+] }
```

Comment:
Personally, I don't like `sqlx`.
- It's heavy
- It has features which do requests to DB at compile-time via proc macros and that's **awful** in many ways (even if you don't use it)

I would prefer using a different crate, like [rustsqlite](https://crates.io/crates/rusqlite) for example

Reply:
It's synchronous AFAICT...

Reply:
Ahhh, that's unfortunate. Is `sqlx` really asynchronous or is it just using `spawn_blocking`?

I still don't like `sqlx`, but if there is nothing better at the moment, I think we should merge this :shrug: